Container Security/eks-implementation-guide.md

```diff
@@ -144,36 +165,64 @@ Various command-line utilities are required for this demo. The utilities can eit
    }
    ```
 
- - Note the URI of the newly created repository to the environment variable for further use. Falcon Container Sensor will be available under this URI.
+ - Note the `repositoryUri` of the newly created repository to the environment variable for further use. Falcon Container Sensor will be available under this URI.
+
+
+### Step 3: Push the falcon sensor image to the Repository
+
+ - Set the FALCON_IMAGE_URI variable to your managed ECR based on the ECR `repositoryUri`
+ 
+ - Note for existing ECR registries the `repositoryUri` can be found with the following command.
+   ```
+   aws ecr describe-repositories
    ```
+   Example
```

Comment:
```suggestion
   Example output:
```

Reply:
This one is actually an example. I will add the colon for consistency.